src/assets/fonts/fonts.css

```diff
@@ -0,0 +1 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600&display=swap');
```

Comment:
We should store the fonts locally. If you don't want to add font files to this starter just leave a comment here about how to add fonts properly